xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -5153,3 +5153,8 @@ void CVideoPlayer::SetUpdateStreamDetails()
 {
   m_messenger.Put(new CDVDMsg(CDVDMsg::PLAYER_SET_UPDATE_STREAM_DETAILS));
 }
+
+void CVideoPlayer::TriggerCheckPassthrough()
+{
+  m_VideoPlayerAudio->SendMessage(new CDVDMsg(CDVDMsg::PLAYER_AUDIO_DEVICE_CHANGED), 1);
```

Comment:
m_VideoPlayerAudio is not supposed to be accessed fro the outside world.

Reply:
VideoPlayer => VideoPlayerAudio   These part has not changed from initial PR....

Only moved from `CVideoPlayer::OnResetDisplay()`  to  `CVideoPlayer::TriggerCheckPassthrough()`

????

Reply:
CVideoPlayer::OnResetDisplay() is different. It is a cb VP has registered for. Hence VP has full control when this cb can be called.